src/library_idbfs.js

```diff
@@ -15,16 +15,56 @@ mergeInto(LibraryManager.library, {
       // reuse all of the core MEMFS functionality
       return MEMFS.mount.apply(null, arguments);
     },
+#if IDBFS_DEFER_WRITE
+    DEFER_WRITE_TIMEOUT: 10000,
+    deferredSyncFs: function(mount) {
+      // If there is already a timer, don't set another one.
+      if (mount.mountSyncfsTimer) {
+        return;
+      }
+
+      // Set a timer what would trigger syncfs after 10 sec.
+      mount.mountSyncfsTimer = setTimeout(function() {
+        IDBFS.syncfs(mount, false, function(err) {
+          if (err) {
+            console.log("IDBFS.syncfs threw an exception: ", err);
+          }
+        });
+      }, IDBFS.DEFER_WRITE_TIMEOUT);
+
+      // Also set an atExit hook (if we have not yet done so.)
+      if (!mount.atExitSyncFsHookAdded) {
+        mount.atExitSyncFsHookAdded = true;
+        Module['addOnExit'](function() {
+          // We might not need to syncfs() at this time ...
+          if (!mount.mountSyncfsTimer) {
+            return;
+          }
+
+          IDBFS.syncfs(mount, false, function(err) {
+            if (err) {
+              console.log("IDBFS.syncfs threw an exception: ", err);
+            }
+          });
+        });
+      }
+    },
+#endif
     syncfs: function(mount, populate, callback) {
-      IDBFS.getLocalSet(mount, function(err, local) {
+      IDBFS.getRemoteSet(mount, function(err, remote) {
```

Comment:
i'm not sure why that matters? couldn't the clearing be done in the very first line of `syncfs`?